build/common.project.props

```
@@ -124,7 +124,8 @@
     <MinimumVisualStudioVersion>15.0</MinimumVisualStudioVersion>
     <VsixOutputDirName>VS15</VsixOutputDirName>
     <CodeAnalysisRuleSet Condition="'$(CodeAnalysisRuleSet)' == ''">$(BuildCommonDirectory)NuGet.ruleset</CodeAnalysisRuleSet>
-    <EnforceCodeStyleInBuild Condition=" '$(EnforceCodeStyleInBuild)' != '' " >true</EnforceCodeStyleInBuild>
+    <EnforceCodeStyleInBuild Condition="'$(EnforceCodeStyleInBuild)' == ''">true</EnforceCodeStyleInBuild>
+    <WarningsNotAsErrors>$(WarningsNotAsErrors);EnableGenerateDocumentationFile</WarningsNotAsErrors>
```

Shouldn't this be 

```suggestion
    <WarningsNotAsErrors>$(WarningsNotAsErrors);IDE0005</WarningsNotAsErrors>
```

I see a bunch of warnings on the build https://dev.azure.com/dnceng-public/public/_build/results?buildId=683379&view=results

---

That would make it so "Remove unnecessary using directives" won't give an error on build which won't actually enforce that rule. 

But I'm okay with it for this PR since I want to start enabling those types of rules project by project in subsequent PRs.

---

Actually second thought. I switched it to NoWarn, that way IDE0005 can still warn for projects as they are enabled.